src/business-logic/services/rundown-http-service.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpService } from '../interfaces/http-service'
+import { Rundown, RundownMode } from '../../model/entities/rundown'
+import { RundownService } from '../interfaces/rundown-service'
+
+const RUNDOWN_ENDPOINT: string = '/rundowns'
+const BASIC_RUNDOWNS_ENDPOINT: string = `${RUNDOWN_ENDPOINT}/basic`
+
+export class RundownHttpService implements RundownService {
+  public constructor(private readonly httpService: HttpService) {}
+
+  public async getRundown(rundownId: string): Promise<Rundown> {
+    return await this.httpService.get(`${RUNDOWN_ENDPOINT}/${rundownId}`) as Rundown
```

Comment:
I just had a thought :) How would you like to type your HTTP calls like this instead?

```suggestion
    return await this.httpService.get<Rundown>(`${RUNDOWN_ENDPOINT}/${rundownId}`)
```

I made this change to the HttpService to achieve that :)

```
get<T>(url: string): Promise<T>
```